convert-to-pdf.js

```diff
@@ -0,0 +1,30 @@
+const { exec } = require('child_process');
+const PORT = "3000";
+
+const serve = exec("yarn serve", (error, stdout, stderr) => {
+    if (error) {
+      console.error(`exec error: ${error}`);
+      return;
+    }
+    console.log(`${stdout}`);
+    console.error(`${stderr}`);
+});
+setTimeout(() => {
```

Comment:
Let's use [wait-on](https://www.npmjs.com/package/wait-on) instead, and wait until the port is available.  Using a timer will sometimes fail, if the machine doesn't get the server started in time (common on CI builders).